scripts/dev/contexts/root-context

```diff
@@ -46,25 +46,22 @@ export OPS_MANAGER_NAMESPACE="operator-testing-50-current"
 # Moved from the old set_env_context.sh
 export LOCAL_RUN=true
 
-OPERATOR_VERSION=${OPERATOR_VERSION:-"latest"}
-READINESS_PROBE_VERSION=${READINESS_PROBE_VERSION:-"latest"}
-VERSION_UPGRADE_HOOK_VERSION=${VERSION_UPGRADE_HOOK_VERSION:-"latest"}
 if [[ "${OVERRIDE_VERSION_ID:-}" != "" ]]; then
+  echo "Overriding operator version with ${OVERRIDE_VERSION_ID}"
   OPERATOR_VERSION="${OVERRIDE_VERSION_ID}"
-  READINESS_PROBE_VERSION="${OVERRIDE_VERSION_ID}"
```

Comment:
why this change? Wouldn't it remove setting the version when we want to override all images to patch id?